Update the Readme files (#22)
DevExpress-Examples/devextreme-cascading-selectboxes: Readme.md

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E5000)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# SelectBoxes for DevExtreme - How to implement standalone and in-Form cascading SelectBoxes

This example demonstrates how to implement cascading SelectBoxes in the following scenarios:

- Ыtandalone SelectBoxes
- SelectBoxes in Form

> When you select a value from the first SelectBox, the second SelectBox loads the filtered values. In this example, only the cities from the selected state appear in the second SelectBox.

Reviewer: ```suggestion
When you select a value from the first SelectBox, the second SelectBox loads the filtered values. In this example, only cities from the selected state appear in the second SelectBox.
```

> When you select a value from the first SelectBox, the second SelectBoxloads the filtered values. In this example, when you select a state only the cities from this state appear in the second SelectBox.

Reviewer: `the second SelectBoxloads the filtered values` Do you mean 'the second SelectBox loads the filtered values'?

```suggestion
When you select a value from the first SelectBox, the second SelectBox loads the filtered values. In this example, only the cities from the selected state appear in the second SelectBox.
```

![Implement standalone and in-Form cascading SelectBoxes](devextreme-cascading-selectboxes.png)

> To implement this, get a [value](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#value) from the first editor and [filter](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/Methods/#filter) the [dataSource](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#dataSource) by this **value** in the second editor so it shows relevant items.

Reviewer: ```suggestion
To implement this, get the [value](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#value) from the first editor and [filter](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/Methods/#filter) the [dataSource](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#dataSource) by this **value** in the second editor.
```

`To implement this` What does **this** refer to?
